projects/angular/src/layout/API.md

```diff
@@ -0,0 +1,55 @@
+# Card - Style API
+
+## CSS Variables
+
+| CSS Variable Name              | Description                                                                              |
+| ------------------------------ | ---------------------------------------------------------------------------------------- |
+| --clr-card-bg-color            | Changes the background color of card.  |
+| --clr-card-divider-color       | Changes the divider color of the card.                              |
+| --clr-card-title-color         | Changes the text color of card header.           |
+| --clr-card-title-font-weight      | Changes the thickness of card title.                        |
+| --clr-card-border-width | Changes the border width of the card. |
+| --clr-card-border-radius | Changes the border radius of card. |
+| --clr-card-border-color | Changes the border color of the card. |
+| --clr-card-box-shadow-color | Changes the background shadow color of card. |
+| --clr-card-clickable-border-color |  Changes the background color of card when clicked.|
```

Comment:
It should not be "when clicked", more like if the card is clickable. Some cards behave like buttons.